iis-setup.sh

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Define environment variables
-export BENDER=bender-0.27.1
+export BENDER=bender-0.28.1
 export CC=gcc-9.2.0
```

Comment:
This most likely has to be updated also in the `Dockerfile`.

Reply:
I will update bender in a separate PR. I reverted all the related changes, since they are not strictly needed for this PR.